pkg/distributor/forwarding/forwarding.go

```diff
@@ -84,6 +86,11 @@ func NewForwarder(reg prometheus.Registerer, cfg Config) Forwarder {
 			Name:      "distributor_forward_requests_total",
 			Help:      "The total number of requests the Distributor made to forward samples.",
 		}, []string{"user"}),
+		errorsTotal: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Namespace: "cortex",
+			Name:      "distributor_forward_errors_total",
+			Help:      "The total number of errors which the Distributor received from forwarding targets.",
```

Comment:
```suggestion
			Help:      "The total number of errors that the distributor received from forwarding targets.",
```

Reply:
There is a bit of language ambiguity here. "From" can mean two things:

* From targets that performed "forwarding"
* as a result of

I assume you mean the first one.

Reply:
> I assume you mean the first one.

Correct

Reply:
I'll update it to 

```
The total number of errors that the distributor received from forwarding targets when trying to send samples to them.
```